setup.js

```diff
@@ -0,0 +1,108 @@
+import { Nation } from "./constants";
+
+const error = (want) => (x) => {
+  throw new Error(`got=${x.value}, want=${want}`);
+};
+
+export default ({ players }) => {
+  const nationAssignments = {
+    3: ({ id, nation }) =>
+      nation.when({
+        AH: () => [
+          { id, nation: Nation.AH },
+          { id, nation: Nation.GB },
+        ],
+        IT: () => [
+          { id, nation: Nation.IT },
+          { id, nation: Nation.RU },
+        ],
+        FR: () => [
+          { id, nation: Nation.FR },
+          { id, nation: Nation.GE },
+        ],
+        GB: error("AH|IT|FR"),
+        RU: error("AH|IT|FR"),
+        GE: error("AH|IT|FR"),
+      }),
+    2: ({ id, nation }) =>
```

Comment:
Should we swap the positions of `2` and `3` so that they run 2-6 in order? Suuuuper small comment 😄

Reply:
It is done!